contracts/contracts/interfaces/IVault.sol

```diff
@@ -98,6 +98,8 @@ interface IVault {
 
     function rebase() external returns (uint256);
 
+    function totalValue() external view returns (uint256 value);
```

Comment:
did you intend to make this change in the interface?

Reply:
I was using it for testing, but then figured I might as well leave it in.